### DISCUSSION TITLE
What all languages this package support?

### DISCUSSION THREAD

**Discussion by abhishek5324** [nlp-compromise/nlp_compromise#902] [Q&A] [answered]
Hello @spencermountain. This package is awesome. Can you tell me what all languages does this package support?

**Comment by spencermountain** [comment, marked as answer]
thanks @abhishek5324 [french](https://github.com/nlp-compromise/fr-compromise) [spanish](https://github.com/nlp-compromise/es-compromise) and [german](https://github.com/nlp-compromise/de-compromise) right now

**Reply by abhishek5324** [reply]
That's great. Can you tell me how can I use those languages? It would be really helpful. Thanks @spencermountain
